### PR TITLE
Improve mantid conda installation instructions

### PR DESCRIPTION
**Pull request by robertapplin**
Improve mantid conda installation instructions

For now, I'm hesitant to add instructions about using mamba for fear of information overload

### CODE REVIEW

File: source/installation/index.rst [hunk 1]
```diff
@@ -59,15 +59,28 @@ channel on Anaconda.org. The available packages are:
...
+It is worth noting that versions of Mantid after v6.7 (including the nightly) use
+Python 3.10. Mantid v6.7 and before use Python 3.8.
+
 Latest Release
 ##############
 
-To install any of the packages simply use the `-c` option in Conda to add the
-mantid channel and then the package name:
+To install Mantid into a new Conda environment use these options:
+- `-n` to specify the name of your new environment
```
Comment (MialLewis) [inline source/installation/index.rst:69]: If you want this to be a bullet point list, I think this needs a whiteline above (at least it isn't showing on the preview!)

**Review by MialLewis — DISMISSED** [review]
One comment to address.

I think it's worth having the discussion about linking to a `mamba` install guide page, as currently installing mantid with `conda` takes so long many may abandon as it feels like it's not working.

**Comment by robertapplin** [comment]
> One comment to address.
> 
> I think it's worth having the discussion about linking to a `mamba` install guide page, as currently installing mantid with `conda` takes so long many may abandon as it feels like it's not working.

Do you have a link in mind I can use?

**Review by MialLewis — APPROVED** [review]
LGTM, can you raise an issue to discuss how best to encourage users to use mamba?

**Comment by robertapplin** [comment]
> LGTM, can you raise an issue to discuss how best to encourage users to use mamba?

Opened an issue here https://github.com/mantidproject/mantid/issues/36178

**Pull request merged**